lib/png12-0.xml

```diff
@@ -8,8 +8,35 @@ PNG (Portable Network Graphics) format files.
 
 This package contains the runtime library files needed to run software
 using libpng.</description>
+  <homepage>http://gnuwin32.sourceforge.net/packages/libpng.htm</homepage>
```

Comment:
I'd suggest not putting the gnuwin32 homepage here, since this feed also contains Linux builds.

Reply:
used http://libpng.org/pub/png/libpng.html instead